core/src/main/java/co/cask/wrangler/steps/transformation/Quantization.java

```diff
@@ -55,7 +53,9 @@ public Quantization(int lineno, String detail, String col1, String col2, String
     while(matcher.find()) {
       double lower = Double.parseDouble(matcher.group(1));
       double upper = Double.parseDouble(matcher.group(2));
-      rangeMap.put(Range.closed(lower, upper), matcher.group(3));
+      rangeMap.put(lower, matcher.group(3));
+      // store null in place of upper bound
+      rangeMap.put(upper, null);
```

Comment:
don't you need to change the way the map is searched? (in line 92)
I feel that it would be better to introduce a helper class that replaces RangeMap.